projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-detail-items/order-detail-items.component.html

```diff
@@ -45,7 +45,7 @@
       </div>
       <div class="cx-list-item col-12">
         <cx-cart-item-list
-          [items]="getConsignmentProducts(consignment)"
```

Comment:
Is the ` getConsignmentProducts` still used?  Otherwise perhaps it should be deprecated.

Reply:
...it's already deprecated.